packages/app-desktop/gui/StatusScreen/StatusScreen.tsx

```diff
@@ -47,9 +49,10 @@ function StatusScreen(props: Props) {
 		setReport(r);
 	}
 
+
 	useEffect(() => {
 		void resfreshScreen();
-	}, []);
+	}, [props.decryptionWorkerState]);
```

Comment:
Do you know why `decryptionWorkerState` is needed here? Is that maybe some parameter we should pass to ReportService? If possible it's best if can avoid implied dependencies like this